index.js

```diff
@@ -77,6 +77,34 @@ function validatePaperWalletMnemonic(input) {
   }
 }
 
+function validateAddressType(input, allowed_min, allowed_max) {
+  // BASE_ADDRESS_KEY_KEY = const(0)
```

Comment:
those commented out constants seem like they could be an "enum" - given that we are not using Flow/Typescript here I would keep it simple and have it as a simple objec